src/main/java/com/osp/dragon/service/ArticlesReader.java

```diff
@@ -0,0 +1,35 @@
+package com.osp.dragon.service;
+
+import com.osp.dragon.model.Article;
+import org.springframework.stereotype.Service;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class ArticlesReader {
+
+    private static final String FILE_NAME = "C:/Users/Raczkowski/IdeaProjects/dragon2/src/main/resources/articles.csv";
```

Comment:
to jest ok, ale na linuxie czyli na serwerze nie odpalisz bo nie ma takiej ścieżki

Reply:
a tutaj nie wiem, bo jak dam zamiast ścieżki nazwe pliku to nie czyta artykułów